### DISCUSSION TITLE
背景水印图分享

### DISCUSSION THREAD

**Discussion by Z-fly** [lanyeeee/bilibili-manga-watermark-remover#49] [General]
[水印.zip](https://github.com/user-attachments/files/17662657/default.zip)

**Comment by lanyeeee** [comment]
首先，图片是很大的资源，不可能内置在程序里，会极大增加程序体积

其次，即使是 `同一本漫画` + `同一个尺寸`，也会存在不同的水印，从而导致去水印失败，详见 #8
而用预制的水印图相当于 `不同的漫画` + `同一个尺寸`，失败的概率就更高了

如果要分享可以考虑开个discussion，也许会帮到需要的人
当然我个人还是建议用同一本漫画里的图片来生成对应的`背景水印图`，这是成功率最高的做法

~~其实如果漫画生成不了`背景水印图`，用尺寸差不多的背景水印图拉伸到漫画尺寸也能去水印，只是会留痕，虽然眼睛不太看得出来~~
